android/src/main/java/com/swmansion/gesturehandler/core/GestureHandlerOrchestrator.kt

```diff
@@ -702,5 +709,8 @@ class GestureHandlerOrchestrator(
       state == GestureHandler.STATE_CANCELLED ||
         state == GestureHandler.STATE_FAILED ||
         state == GestureHandler.STATE_END
+
+    private fun isActive(state: Int) =
+      state == GestureHandler.STATE_ACTIVE
```

Comment:
I don't think it is used anywhere 😅

Reply:
removed [ef986a0](https://github.com/software-mansion/react-native-gesture-handler/pull/3095/commits/ef986a0f4c8888e11bff94436f738276ec57ef07)